app/javascript/mastodon/components/empty_account.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DisplayName from './display_name';
+import Skeleton from 'mastodon/components/skeleton';
+import classNames from 'classnames';
+
+const EmptyAccount = ({ size, minimal }) => {
+  return (
+    <div className={classNames('account', { 'account--minimal': minimal })}>
+      <div className='account__wrapper'>
+        <div className='account__display-name'>
+          <div className='account__avatar-wrapper'><Skeleton width={size} height={size} /></div>
+
+          <div>
+            <DisplayName />
+            <Skeleton width='7ch' />
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+EmptyAccount.propTypes = {
+  size: PropTypes.number,
+  minimal: PropTypes.bool,
+};
+
+EmptyAccount.defaultProps = {
+  size: 46,
```

Comment:
I'm a bit concerned about multiplying the places where we define the same default value.

Reply:
We could definitely do something were EmptyAccount inherits the default prop for size from Account?

Reply:
Hm, let's leave that out for now.

Reply:
I've converted this to typescript, and made it a default prop, so same thing just in typescript.